cns/restserver/ipam_test.go

```diff
@@ -1717,3 +1727,170 @@ func TestIPAMGetK8sSWIFTv2IPFailure(t *testing.T) {
 		t.Fatal("Expected available ips to be 2 since we expect the IP to not be assigned")
 	}
 }
+
+//
+//func TestIPAMGetSFSWIFTv2IPSuccess(t *testing.T) {
+//	req := cns.IPConfigsRequest{
+//		PodInterfaceID:   testPod1Info.InterfaceID(),
+//		InfraContainerID: testPod1Info.InfraContainerID(),
```

Comment:
remove these UTs

Reply:
will we need these UTs in the future? if not then we should remove it instead of commenting them out

Reply:
no i do plan to remove them